github/actions.go

```
@@ -104,7 +105,7 @@ func (c *WorkflowClient) EnrichWorkflowRunsWithParams(ctx context.Context, filte
 	for _, run := range runs {
 		params, err := c.FetchWorkflowRunParams(ctx, filter, run.JobRunID)
 		if err != nil {
-			return err
+			log.Warn(fmt.Sprintf("Failed fetching workflow params for workflow: %v runId: %d, it will be ommitedd, err: %v\n", run.WorkflowName, run.JobRunID, err))
```

I was thinking whether or not to implement the same behaviour as in server-mod. The reason I did not do it was because I didn't want to show logs when the tool is used in a non server mode because this would prevent chaining the tool with linux pipes for example `github-workflow-dashboard -repo xxx -owner yyy -format json | json_pp`. However at the same time I din't want to silently ignore the error.

We can merge this as it is and later change it to verbose logging: https://github.com/NewestUser/github-workflow-dashboard/issues/6#issue-1151845050

I guess I can introduce a verbose flag that would show more logs. 

PS: new line will be added by the logger, you can remove the `\n` at the end of the Sprintf

---

Thank you for checking. The point above makes sense (I'm sorry if I opend the PR in the not-good timing).
I've removed the `\n` and bumped the version (also included the updated binary).

---

Timing is great. Thanks for the PR!